packages/castmill/lib/castmill/resources.ex

```diff
@@ -706,17 +706,39 @@ defmodule Castmill.Resources do
     with an existing entry.
   """
   def add_channel_entry(channel_id, entry_attrs \\ %{}) do
```

Comment:
Is there some thread safe mechanism in place that guarantees that no overlapping entries are added to the same channel?

Reply:
No, it is not actually supported. I tried to get a solution using grok but it went bananas... we will have to fix it in a separate PR as it is a bit more involved than I thought.